client/submitAssignment.js

```diff
@@ -4,7 +4,11 @@ let validCompileCmd = false;
 let supportedLanguage = "";
 let cmdArgsEnabled = false;
 
-// Gets the supported language for current environment from the server.
+/**
```

Comment:
This doesn't follow the same multi-line comment style as followed in the previous comment.
Previously followed.
```
/*
comment
comment
...
*/
```

Here.
```
/*
* comment
* comment
* ...
*/
```

Follow the accepted de-factor format for multi-line comments in javascript.

Reply:
The one followed here is the format that javascript provides for JSDoc, similar to goDoc. The one previously followed is just the multi line comment.

Reply:
I don't comprehend. Why the difference in multi-line comment formats for constants.js and submitAssignment.js?

Reply:
Kept the multi line comment style consistent